common/lib/types/protocolmessage.js

```diff
@@ -66,5 +66,47 @@ var ProtocolMessage = (function() {
 		return Utils.mixin(new ProtocolMessage(), values);
 	};
 
+	ProtocolMessage.actionName = function(index) {
+		for (var needle in ProtocolMessage.Action) {
+			if (ProtocolMessage.Action[needle] === index) {
+				return needle;
+			}
+		}
+	}
+
+	function toStringArray(array) {
+		var result = [];
+		if (array) {
+			for (var i = 0; i < array.length; i++) {
+				result.push(array[i].toString());
+			}
+		}
+		return '[ ' + result.join(', ') + ' ]';
+	}
+
+	ProtocolMessage.stringify = function(msg) {
```

Comment:
Because this a class method, not an instance method.  We are using the `ProtocolMessage` class method to convert a JSON object into a string representation.